src/website/app/Router.js

```diff
@@ -54,6 +57,25 @@ export default function Router({ demos }: Props) {
     <Switch>
       {routes}
       <Route
+        path="/components/:componentId/:exampleId"
+        render={route => {
+          const componentId = route.match.params.componentId;
```

Comment:
You could destructure both componentId and exampleId here and then reference them thereafter.

Reply:
👍